### ISSUE TITLE
gRPC SLOs

### ISSUE THREAD

**Issue by kakkoyun** [metalmatze/slo-libsonnet#15]
Would you consider adding similar SLO for gRPC based services?

**Comment by metalmatze** [comment]
Yes, definitely! Next to http based SLOs I'm super happy to add gRPC! :+1:

**Comment by brancz** [comment]
Any new developments on this?

**Comment by brancz** [comment]
fwiw with #44 I was able to do gRPC SLOs :)
